dev-libs/ocl-icd/ocl-icd-2.2.12.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby23 ruby24 ruby25"
+
+inherit ruby-single multilib-minimal
+
+DESCRIPTION="Alternative to vendor specific OpenCL ICD loaders"
+HOMEPAGE="https://forge.imag.fr/projects/ocl-icd/"
+SRC_URI="https://forge.imag.fr/frs/download.php/836/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```

Comment:
Empty variable can be omitted.

Reply:
I usually keep empty IUSE. no reason behind it)